egs/mgb2/asr/local/run_test_mgb.sh

```diff
@@ -0,0 +1,111 @@
+#! /bin/bash
```

Comment:
We do not have to include this.
Please remove it.

Reply:
done